pkgconf-3.0.0 API update

https://bugs.gentoo.org/974972

--- a/src/external/pkgconfig_libpkgconf.c
+++ b/src/external/pkgconfig_libpkgconf.c
@@ -19,6 +19,7 @@
 #include "log.h"
 #include "options.h"
 #include "platform/filesystem.h"
+#include "platform/os.h"
 #include "platform/path.h"
 #include "tracy.h"
 
@@ -38,2 +39,11 @@
 
+#if defined(LIBPKGCONF_VERSION) && LIBPKGCONF_VERSION >= 30000
+static const char *
+environ_lookup_handler(const pkgconf_client_t *client, const char *key)
+{
+	(void)client;
+	return os_get_env(key);
+}
+#endif
+
 static bool
@@ -41,7 +51,11 @@
 {
 	TracyCZoneAutoS;
 	c->personality = pkgconf_cross_personality_default();
+#if defined(LIBPKGCONF_VERSION) && LIBPKGCONF_VERSION >= 30000
+	pkgconf_client_init(&c->client, error_handler, NULL, c->personality, NULL, environ_lookup_handler);
+#else
 	pkgconf_client_init(&c->client, error_handler, NULL, c->personality);
+#endif
 
 	struct obj_array *pkg_config_path;
 	{
