--- a/lib/libimhex/include/hex/helpers/http_requests_native.hpp
+++ b/lib/libimhex/include/hex/helpers/http_requests_native.hpp
@@ -4,6 +4,7 @@
 
     #include <string>
     #include <future>
+    #include <mutex>
 
     #include <curl/curl.h>
 
--- a/lib/libimhex/include/hex/helpers/types.hpp
+++ b/lib/libimhex/include/hex/helpers/types.hpp
@@ -2,6 +2,7 @@
 
 #include <cstddef>
 #include <cstdint>
+#include <type_traits>
 
 #include <concepts>
 
