--- a/lib/external/libwolv/libs/math_eval/source/math_eval/math_evaluator.cpp
+++ b/lib/external/libwolv/libs/math_eval/source/math_eval/math_evaluator.cpp
@@ -0,0 +1 @@
+#include <cstdlib>
--- a/lib/libimhex/include/hex/api/task_manager.hpp
+++ b/lib/libimhex/include/hex/api/task_manager.hpp
@@ -0,0 +1 @@
+#include <atomic>
--- a/lib/libimhex/source/helpers/udp_server.cpp
+++ b/lib/libimhex/source/helpers/udp_server.cpp
@@ -0,0 +1 @@
+#include <vector>
--- a/lib/third_party/HashLibPlus/Utils/Converters.h
+++ b/lib/third_party/HashLibPlus/Utils/Converters.h
@@ -0,0 +1 @@
+#include <cstdlib>
