https://bugs.gentoo.org/945828

--- a/configure.ac
+++ b/configure.ac
@@ -26,7 +26,11 @@
 AC_SUBST([GETTEXT_PACKAGE])
 AM_SILENT_RULES([yes])
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.18.2])
+dnl Support for _REQUIRE_VERSION was added in gettext 0.19.6. If both
+dnl _REQUIRE_VERSION and _VERSION are present, the _VERSION is ignored.
+dnl We use both for compatibility with other programs in the Autotools family.
+AM_GNU_GETTEXT_REQUIRE_VERSION([0.19.6])
+AM_GNU_GETTEXT_VERSION([0.19.6])

 m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 AX_REQUIRE_DEFINED([AX_ADD_FORTIFY_SOURCE])
